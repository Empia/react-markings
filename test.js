// @flow
'use strict';

const md = require('./');
const React = require('react');
const {renderToStaticMarkup} = require('react-dom/server');

function MyComponent() {
  return React.createElement('div', null, 'MyComponent');
}

test('example', () => {
  expect(renderToStaticMarkup(md`
    # Heading

    ${React.createElement(MyComponent)}
  `)).toMatchSnapshot();
});
