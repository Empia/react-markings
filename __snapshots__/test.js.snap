// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`paragraph 1`] = `"<div><h1>Heading</h1><div>MyComponent</div></div>"`;
